lib/trino-hive-formats/src/main/java/io/trino/hive/formats/ReadWriteUtils.java

```diff
@@ -315,4 +317,13 @@ private static int calculateTruncationLength(int maxCharacterCount, Slice slice,
         }
         return indexEnd - offset;
     }
+
+    public static RuntimeException closeAndRethrow(Throwable throwable, Closeable closeable)
```

Comment:
Seems like this method can be replaced with
```java
try (closeable) {
    throw throwable;
}
```
With the advantage that doing it inline won't change the exception types